rocks.syng.Syng.yaml

```diff
@@ -0,0 +1,47 @@
+id: rocks.syng.Syng
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+base: "com.riverbankcomputing.PyQt.BaseApp"
+base-version: "6.7"
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+finish-args:
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess - /app/cleanup-BaseApp.sh
+  # X11 + XShm access
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  # Acceleration
+  - --device=dri
+  # Sound
+  - --socket=pulseaudio
+  # Ability to edit/save files anywhere
+  - --filesystem=host
```

Comment:
anywhere? Really?

Reply:
comment is misleading (will fix), this needs to be set for playback of karaoke files, that could be stored anywhere on the system, will be changed to `--filesystem=host:ro`